incgamma.go

```diff
@@ -0,0 +1,25 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "github.com/gonum/mathext/internal/cephes"
+
+// IncGamma computes the incomplete Gamma integral
+// In this implementation both arguments must be positive.
+// The integral is evaluated by either a power series or
+// continued fraction expansion, depending on the relative
+// values of a and x.
+func IncGamma(a, x float64) float64 {
```

Comment:
perhaps add a link to http://mathworld.wolfram.com/IncompleteGammaFunction.html
or https://en.wikipedia.org/wiki/Incomplete_gamma_function